src/api/app/views/person/token/index.xml.builder

```diff
@@ -1,6 +1,6 @@
 xml.directory(count: @list.length) do |dir|
   @list.each do |token|
-    p = { id: token.id, string: token.string, kind: token.token_name }
+    p = { id: token.id, string: token.string, kind: token.token_name, name: token.name }
```

Comment:
:thought_balloon: `token.token_name` and `token.name` could be a source of confusion

Reply:
I agree :+1: 

I created a card in our backlog for this: https://trello.com/c/0yujk7O5/2391-remove-tokentokenname-method

Reply:
@dmarcoux 

It would be much better to have fixed it here than try to refactor it in the future (spoiler: I'm in the future, and we won't refactor it :))